Sta sample improvements (#9852)
nrfconnect/sdk-nrf: samples/wifi/sta/README.rst

********

This sample can perform Wi-Fi operations such as connect and disconnect in the 2.4GHz and 5GHz bands depending on the capabilities of an access point.
The sample also adds LED support to map with connection and disconnection events:

>     * **LED1** starts blinking when the sample is connected to the access point
>     * **LED1** stops blinking when the sample is disconnected from the access point

Reviewer: ```suggestion
    * **LED1** starts blinking when the sample is connected to the access point.
    * **LED1** stops blinking when the sample is disconnected from the access point.
```

********

This sample can perform Wi-Fi operations such as connect and disconnect in the 2.4GHz and 5GHz bands depending on the capabilities of an access point.
The sample also adds LED support to map with connection and disconnection events:

    * **LED1** starts blinking when the sample is connected to the access point.
    * **LED1** stops blinking when the sample is disconnected from the access point.

Using this sample, the development kit can connect to the specified access point in :abbr:`STA (Station)` mode.

IP addressing
=============

> The sample uses DHCP to obtain an IP address for the Wi-Fi interface, but it starts with a default static IP address
> to handle networks without DHCP servers, or if the DHCP server is not available. Successful DHCP handshake will override
> the default static IP configuration.

Reviewer: ```suggestion
The sample uses DHCP to obtain an IP address for the Wi-Fi interface.
It starts with a default static IP address to handle networks without DHCP servers, or if the DHCP server is not available.
Successful DHCP handshake will override the default static IP configuration.
```

> The default static configuration is below and can be changed in the ``prj.conf`` file:

Reviewer: ```suggestion
The following static configuration is by default and can be changed in the :file:`prj.conf` file:
```
